Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM redash/redash:latest
+
+COPY . /redash-stmo
+
+ENV PYTHONUNBUFFERED=0 \
+	PYTHONPATH=/app/ \
+	REDASH_LOG_LEVEL="INFO" \
+	REDASH_REDIS_URL=redis://redis:6379/0 \
+	REDASH_DATABASE_URL=postgresql://postgres@postgres/postgres
+
+RUN pip install /redash-stmo
```

Comment:
This will need to be `pip install -e /redash-stmo` (or `--editable`) so the volume mounted under /redash-stmo in development will pick up changes done to the code base.

What `-e` does is create an "egg-link" file which is just a NIH version of a symlink that is picked up automatically when setuptools is installed.